src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewRenderer.cs

```diff
@@ -592,6 +589,7 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 					return;
 				}
 
+				_child.PlatformView.Layout(0, 0, widthMeasureSpec, heightMeasureSpec);
```

Comment:
This cannot possibly work; `widthMeasureSpec` and `heightMeasureSpec` are not integer values for width and height. They are logically ORed with the measureSpec mode. Trying to use them as values for Right and Bottom will give you hugely negative values in many cases.

Also, calling Layout during OnMeasure makes no sense; the values you'd use for Layout don't exist yet because you haven't measured the View.

Reply:
I think you are correct that the implementation of `OnLayout` was wrong - instead of 

```csharp
_child.PlatformView.Layout(l, t, r, b);
```

it should probably be 

```csharp
_child.PlatformView.Layout(0, 0, r - l, b - t);
```

and `OnMeasure` should be left as it was.

Reply:
You are right. The new layout calculation do fix the problem. I have just tested it. Update the code now.